app/javascript/components/Results/Results.css

```diff
@@ -312,3 +325,6 @@
   }
 }
 
+
+
+
```

Comment:
Can we remove this whitespace? LGTM otherwise.

Reply:
Updated.